.vscode/launch.json

```diff
@@ -2,14 +2,21 @@
   // Use IntelliSense to learn about possible attributes.
   // Hover to view descriptions of existing attributes.
   // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+
+  // NOTE: --disable-extensions
+  // Disable all installed extensions for increas performance of playground 
+  // and prevent potential problem and collision with developer's extansion
```

Comment:
```suggestion
   // Disable all installed extensions to increase performance of the debug instance 
   // and prevent potential conflicts with other installed extensions.
```

Reply:
In this repo we have duplicate of comment